frontend/packages/console-app/src/components/network-policies/create-network-policy.tsx

```diff
@@ -8,15 +8,16 @@ import './_create-network-policy.scss';
 
 export const CreateNetworkPolicy: React.FunctionComponent<{}> = () => {
   const { t } = useTranslation();
-  const namespace = getActiveNamespace();
-
+  const namespaceProps = {
+    namespace: getActiveNamespace(),
+  };
   return (
     <div className="co-m-pane__body co-m-pane__form">
       <h1 className="co-m-pane__heading co-m-pane__heading--baseline">
         <div className="co-m-pane__name">{t('public~Create NetworkPolicy')}</div>
         <div className="co-m-pane__heading-link">
           <Link
-            to={`/k8s/ns/${namespace}/networkpolicies/~new`}
+            to={`/k8s/ns/${namespaceProps.namespace}/networkpolicies/~new`}
```

Comment:
I would build the base URL with `resourcePathFromModel` then append `/~new`. (Eventually we'll add a `cluster` path section, so better to use the utility.)